src/components/MapEditor/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { memo , useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Map, Marker } from '@datapunt/react-maps';
+import { ViewerContainer } from '@datapunt/asc-ui';
+import { Zoom } from '@datapunt/amsterdam-react-maps/lib/components';
+import styled from '@datapunt/asc-core';
+import MAP_OPTIONS from 'shared/services/configuration/map-options';
+import BackgroundLayer from 'components/BackgroundLayer';
+import { markerIcon } from 'shared/services/configuration/map-markers';
+import { feature2location } from 'shared/services/map-location';
+
+
+const MapWrapper = styled.div`
+  position: relative;
+`;
+
+const StyledMap = styled(Map)`
+  width: 100%;
+  height: 450px;
+`;
+
+const StyledViewerContainer = styled(ViewerContainer)`
+  z-index: 400;
+`;
+
+const MapEditor = ({ location }) => {
```

Comment:
This name is a bit confusing. Reading the component name gives me the idea that this component is an editor instead of a map

Reply:
I don't like it either, I'm glad to change it please make a suggestion

Reply:
Why not just `Map`?